frontend/src/Pages/ClassEnrollmentDetails.tsx

```diff
@@ -28,6 +29,8 @@ export default function ClassEnrollmentDetails() {
     const { class_id } = useParams<{ class_id: string }>();
     const navigate = useNavigate();
     const { redirect } = useLoaderData() as ClassLoaderData;
+    const { class: clsInfo } = useLoaderData() as ClassLoaderData;
+    const thisSelectedClasStatus = clsInfo?.status;
```

Comment:
@corypride Is this necessary? I can see if you were to create this variable it could hold the expression itself (e.g., `const blockEdits = clsInfo?.status === SelectedClassStatus.Cancelled || clsInfo?.status === SelectedStatus.Completed`)